application/feature/characters/detail/api/src/main/java/com/stepango/blockme/feature/characters/detail/api/domain/model/ICharacterDetail.kt

```diff
@@ -1,8 +1,5 @@
 package com.stepango.blockme.feature.characters.detail.api.domain.model
 
-interface ICharacterDetail {
-    val id: Long
-    val name: String
-    val description: String
-    val imageUrl: String
-}
\ No newline at end of file
+import com.stepango.blockme.feature.characters.core.api.domain.model.ICharacter
+
+interface ICharacterDetail : ICharacter
```

Comment:
For details we can use ICharacter type too, cause it doesn't have unique entity logic. 
_(it will be not scalable if details will be extended, but for our sample it doesn't need)_